docs/reference/sql/limitations.asciidoc

```diff
@@ -147,8 +147,8 @@ The query above is possible because it is equivalent with:
 include-tagged::{sql-specs}/docs/docs.csv-spec[limitationSubSelectRewritten]
 --------------------------------------------------
 
-But, if the sub-select would include a `GROUP BY` or `HAVING` or the enclosing `SELECT` would be more complex than `SELECT X
-FROM (SELECT ...) WHERE [simple_condition]`, this is currently **un-supported**.
+Sub-selects that require the aggregation of already aggregated data are currently **not supported**. For example,
+`SELECT SUM(c) FROM (SELECT COUNT(*) c FROM emp)` fails because it results in a nested aggregation.
```

Comment:
I wouldn't change the previous message. We don't know the full extent of sub-selects that are not supported and this is in general a grey area. Rather than saying we don't support X and Y and (and implicitly leaving out a possible unsupported scenario Z), I would keep the more generic initial statement.

Reply:
makes sense, but I also wanted to highlight the problem with nested aggregations. Because the query `SELECT SUM(c) FROM (SELECT COUNT(*) c FROM emp)` does neither have a `GROUP BY` nor a `HAVING` but is still unsupported. I'll try to rephrase it more broadly.

Reply:
That's the implicit GROUP BY. Imo, this is not a counter-argument.